java-compute/google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/Util.java

```diff
@@ -19,7 +24,7 @@ public static void cleanUpComputeInstances(
     ListPagedResponse listPagedResponse = instancesClient.list(project, zone);
     for (Instance instance : listPagedResponse.iterateAll()) {
       if (isCreatedBeforeThresholdTime(
-              ZonedDateTime.parse(instance.getCreationTimestamp()).toInstant())
+          ZonedDateTime.parse(instance.getCreationTimestamp()).toInstant())
           && instance.getName().startsWith(BaseTest.COMPUTE_PREFIX)) {
```

Comment:
Going to pass in the prefix into function in a separate PR. https://github.com/googleapis/google-cloud-java/pull/8260#discussion_r968578289